jobsAdmin/JobsAdmin.go

```diff
@@ -238,13 +238,12 @@ func (ja *jobsAdmin) recordTuningCompleted(showOutput bool) {
 // There will be only 1 instance of the jobsAdmin type.
 // The coordinator uses this to manage all the running jobs and their job parts.
 type jobsAdmin struct {
-	atomicSuccessfulBytesInActiveFiles int64
```

Comment:
atomicSuccessfulBytesInActiveFiles is removed here.

Reply:
this is unused in the code thats why it was removed.